lfbb_cpp/inc/lfbb_impl.hpp

```diff
@@ -35,7 +35,12 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
  * This file is part of LFBB_CPP - Lock Free Bipartite Buffer
- *
+ * Define LFBB_TRACE and LFBB_FORMATTER to get debug tracing.
```

Comment:
The explanation of usage would go into README.

Reply:
Done.